pulpcore/cli/common/context.py

```diff
@@ -61,10 +48,6 @@ class PluginRequirement(NamedTuple):
     inverted: bool = False
 
 
-# This is deprecated we keep it to not break plugins immediately
-new_component_names_to_pre_3_11_names: Dict[str, str] = {}
```

Comment:
Same here, have all CLI plugins updated to not use this anymore?

Reply:
I'll check.